tests/python/topi/python/test_topi_sparse.py

```diff
@@ -73,12 +68,12 @@ def check_device(device):
         assert a.data.dtype == A.data.dtype
         assert a.indices.dtype == A.indices.dtype
         assert a.indptr.dtype == A.indptr.dtype
-        f = tvm.build(s, [nr, A.data, A.indices, A.indptr, B, C, D], device, name="csrmv")
+        f = tvm.build(s, [nr, A.data, A.indices, A.indptr, B, C, D], target, name="csrmv")
         f(_nr, a.data, a.indices, a.indptr, b, c, d)
         tvm.testing.assert_allclose(d.numpy(), d_np, rtol=1e-4, atol=1e-4)
 
-    for device in ["llvm"]:
-        check_device(device)
+    for target, dev in tvm.testing.enabled_targets():
+        check_device(target, dev)
```

Comment:
Seems the schedules for sparse ops are not ready for GPU? I see the CI failed in cases like this.
We can add a TODO notes here.

Reply:
Yes agreed. there's a few of them like that which needs to be fixed. WIP.